docs/rules/require-prop-comment.md

```diff
@@ -0,0 +1,180 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/require-prop-comment
+description: require that props have a comment
+---
+# vue/require-prop-comment
+
+> require that props have a comment
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+
+## :book: Rule Details
+
+This rule enforces that every prop has a comment that documents it.
+
+<eslint-code-block :rules="{'vue/require-prop-comment': ['error']}">
+
+```vue
+<!-- ✓ GOOD -->
+<template>
+  <div>1</div>
+</template>
+<script>
+import { defineComponent } from '@vue/composition-api'
+
+export default defineComponent({
+  props: {
+    /**
+     * a comment
+     */
+    a: Number,
+    // b comment
+    b: Number,
+    // c
+    // comment
+    c: Number
+  }
+})
+</script>
+```
+
+</eslint-code-block>
+
+<eslint-code-block :rules="{'vue/require-prop-comment': ['error']}">
+
+```vue
+<!-- ✗ BAD -->
+<template>
+  <div>1</div>
+</template>
```

Comment:
Again:

```suggestion
```

Reply:
This test is to check block，default is JSDoc

Reply:
Completed